src/utils.ts

```diff
@@ -152,6 +165,27 @@ export function createSegments(bucket: any, mainColor: string, isHighlight: bool
     }));
 }
 
+
+
+/**
+ * Creates a timeseries data for facet sparkline.
+ *
+ * @export
+ * @param {any[]}           sparklineXDomain A domain of x values of the sparkline data.
+ * @param {Object}          sparklineData    Sparkline data that contains facet instacne and highlight count that mapped to x values.
```

Comment:
typo & grammar: should probably be "Sparkline data that contains facet instance and highlight counts mapped to x values."